client/src/pages/SignUp.js

```diff
@@ -0,0 +1,192 @@
+import React, { useState, useMemo } from "react";
+import {
+  Grid,
+  Typography,
+  FormControl,
+  TextField,
+  Link,
+  Button,
+} from "@material-ui/core";
+import useStyles from "./styles";
```

Comment:
I'd like the styles to be more specific to each page. Can you make different style objects and add them to the component files themselves? If they get too big, then we can split them out and make more common styles between components

Reply:
Just left this hook inside separate file because both pages share same code. But for other pages will do for each component inside it. I will change the name for this though (like LoginSignupStyles.js or similar).